src/batou/secrets/tests/test_manage.py

```diff
@@ -38,6 +40,32 @@ def test_manage__2(tmp_path, monkeypatch, capsys):
     assert "306151601E813A47" in out
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 7),
+    reason="age is available in tests with python 3.7 only",
+)
+def test_manage__2_age(tmp_path, monkeypatch, capsys):
+    """It allows to add/remove_users in an age encrypted environment."""
+    shutil.copytree("examples/tutorial-secrets", tmp_path / "tutorial-secrets")
+    monkeypatch.chdir(tmp_path / "tutorial-secrets")
+
+    key_name = "https://github.com/ctheune.keys"
+
+    summary()
+    out, err = capsys.readouterr()
+    assert key_name in out
```

Comment:
This smells like it might be a good use case for pytest-patterns. ;) Not sure though.